msteams-platform/messaging-extensions/high-quality-message-extension.md

```diff
@@ -20,7 +20,7 @@ Microsoft 365 plugins provide integration with various  Microsoft 365 products,
 
 We recommend that you build or upgrade your existing message extensions to maximize their usefulness and usability in Copilot for Microsoft 365. Message extensions should support one or more search commands, as these are recognized by Copilot for Microsoft 365 as skills it can execute on behalf of the user. Additionally, your extensions must meet the standards for compliance, performance, security, and user experience outlined in this article.
 
-:::image type="content" source="../assets/images/Copilot/ailib-copilot-interface.png" alt-text="Graphic shows the user experience between Microsoft Teams,Microsoft Copilot for Microsoft 365 and Microsoft 365 Chat.":::
+:::image type="content" source="../assets/images/Copilot/ailib-copilot-interface.png" alt-text="Graphic shows the user experience between Microsoft Teams, Microsoft Copilot for Microsoft 365 and M365 Chat.":::
```

Comment:
This makes it sound like there are 3 experiences. For clarity, how about something like:

Graphic shows the user experience between Microsoft Teams and Copilot for Microsoft 365 (M365 Chat).

Reply:
Updated.